omegaconf/base.py

```diff
@@ -222,6 +217,11 @@ def _get_root(self) -> "Container":
             assert root is not None and isinstance(root, Container)
         return root
 
+    def _is_missing(self) -> bool:
+        ret = self._value() == "???"
```

Comment:
I rebased on top of your `missing` branch (assuming it would be merged first). Now this reads:
```python
   def _is_missing(self) -> bool:
        return _is_missing_value(self)
```